code/modules/urist/modules/jungle/scrap.dm

```diff
@@ -103,16 +103,20 @@
 	name = "debris pile"
 	parts_icon = 'icons/urist/structures&machinery/scrap/vehicle.dmi'
 	loot_list = list(
-		/obj/item/vehicle_part,
-		/obj/item/vehicle_part,
-		/obj/item/vehicle_part,
-		/obj/item/vehicle_part,
+		/obj/item/vehicle_part/random,
+		/obj/item/vehicle_part/random,
+		/obj/item/vehicle_part/random,
+//		/obj/item/vehicle_part/random,
 		/obj/item/stack/rods/scrap,
 		/obj/item/stack/material/plastic/scrap,
 		/obj/item/stack/material/scrap,
 		/obj/item/weapon/material/shard
 		)
 
+/obj/structure/scrap/vehicle/New()
+	if(prob(25))
+		new /obj/structure/vehicle_frame/motorcycle
```

Comment:
HOLY SHIT ..() CALLS AAAAAAAAAAAAAAA

***Forgetting that in New() overrides will actually break shit.***